<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Group Picture Developer Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>391</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutGPDLogin">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelUserName">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditUserName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxStayLoggedIn">
       <property name="text">
        <string>Stay logged in</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
